src/behavior.ts

```diff
@@ -93,4 +101,39 @@ export class VisualBehavior implements IInteractiveBehavior {
             );
         });
     }
+
+    private bindKeyboardEventToDataPoints(): void {
+        this.options.dataPointsSelection.on("keydown", (event: KeyboardEvent, dataPoint: EnhancedScatterChartDataPoint) => {
+            if (event.code !== EnterCode && event.code !== SpaceCode) {
+                return;
+            }
+            this.selectionHandler.handleSelection(dataPoint, event.ctrlKey || event.metaKey);
```

Comment:
why metaKey here?

Reply:
on Mac, users use command key, which is equivalent to metaKey